lib/content-services/src/lib/tag/tags-creator/tags-creator.component.spec.ts

```diff
@@ -295,32 +272,40 @@ describe('TagsCreatorComponent', () => {
             fixture.detectChanges();
             tick(100);
 
-            clickAtHideNameInputButton();
             component.tagNameControlVisible = true;
             fixture.detectChanges();
             tick(100);
 
             expect(getNameInput()).toBe(document.activeElement as HTMLInputElement);
         }));
 
-        it('should be hidden and cleared on discard changes', fakeAsync(() => {
-            component.tagNameControlVisible = true;
-            component.tags = ['Passed tag 1', 'Passed tag 2'];
-            typeTag('test');
-            fixture.detectChanges();
-            tick(100);
-            expect(getNameInput().value).toBe('test');
-
-            component.tagNameControlVisible = false;
-            fixture.detectChanges();
-            tick(100);
-            expect(getNameInput()).toBeFalsy();
+        describe('isNameTagsVisible', () => {
+            it('should return true when tagNameControlVisible is true', () => {
+                component.tagNameControlVisible = true;
+                const result = component.isNameTagsVisible;
+                expect(result).toBeTrue();
+            });
+
+            it('should return false when tagNameControlVisible is false and tags length is 0', () => {
+                component.tagNameControlVisible = false;
+                component.tags = [];
+                const result = component.isNameTagsVisible;
+                expect(result).toBeFalse();
+            });
+        });
 
-            component.tagNameControlVisible = true;
-            fixture.detectChanges();
-            tick(100);
-            expect(getNameInput().value).toBe('');
-        }));
+        describe('isTagsEmpty', () => {
+            it('should return true when tags is not empty', () => {
+                component.tags = ['new tag 1', 'new tag 2'];
+                const result = component.isTagsEmpty;
+                expect(result).toBeTrue();
+            });
+            it('should return false when tags is empty', () => {
+                component.tags = [];
+                const result = component.isTagsEmpty;
```

Comment:
incorrect test, you provide empty tags, but `IsTagEmpty` is expected to be false

Reply:
I have updated the method name and title of the test.